src/vma/main.cpp

```diff
@@ -109,6 +109,8 @@ const char *vma_version_str = "VMA_VERSION: " PACKAGE_VERSION "-" STR(VMA_LIBRAR
 #endif
 			      ;	// End of vma_version_str - used in "$ strings libvma.so | grep VMA_VERSION"
 
+#define VMA_GIT_TITLE		"Git: "
```

Comment:
Please note that banner titles are very short (usually one word) therefore Git: fits here
```
VMA INFO: ---------------------------------------------------------------------------
 VMA INFO: VMA_VERSION: 8.3.1-0 Development Snapshot built on Apr 11 2017 20:04:40
 VMA INFO: Cmd Line: date
 VMA INFO: Current Time: Tue Apr 11 20:05:11 2017
 VMA INFO: Pid: 20736
 VMA INFO: OFED Version: MLNX_OFED_LINUX-3.2-2.0.4.0:
 VMA INFO: Git: e6d6be38765c2989344d4d13bc868cb5b906774b
 VMA INFO: Architecture: x86_64
 VMA INFO: Node: r-aa-fatty28
 VMA INFO: Log Level                      INFO                       [VMA_TRACELEVEL]
 VMA INFO: ---------------------------------------------------------------------------
```